app/helpers/twitter_helper.rb

```diff
@@ -0,0 +1,46 @@
+module TwitterHelper
+  def twitter_enabled
+    if Rails.application.secrets.twitter
+      Rails.application.secrets.twitter['enabled'] == true
+    end
+  end
+
+  def twitter_client
+    Twitter::REST::Client.new do |config|
+      config.bearer_token = Rails.application.secrets.twitter['bearer_token']
+    end
+  end
+
+  def twitter_handle
+    Rails.application.secrets.twitter['twitter_handle']
+  end
+
+  def twitter_user_info
+      $twitter_user_info ||= []
```

Comment:
I've never seen a `$variable`. Is this meant to be an `@` or is it something fancy i am missing?